daemon/network_settings.go

```diff
@@ -4,9 +4,6 @@ import (
 	"github.com/docker/docker/nat"
 )
 
-// FIXME: move deprecated port stuff to nat to clean up the core.
-type PortMapping map[string]string // Deprecated
-
 type NetworkSettings struct {
```

Comment:
oh apparently my work overlapped with @duglin's :) I put this in network/settings.go.

Will remove it.